lib/PuppeteerSharp.Tests/LauncherTests/PuppeteerConnectTests.cs

```diff
@@ -120,6 +120,26 @@ await using (var originalBrowser = await Puppeteer.LaunchAsync(TestConstants.Def
             }
         }
 
+        [PuppeteerTest("launcher.spec.ts", "Puppeteer.connect", "should be able to set browser properties using connect options")]
```

Comment:
This is used to map with pptr tests. we don't need it 
```suggestion
        [PuppeteerTest("launcher.spec.ts", "Puppeteer.connect", "should be able to set browser properties using connect options")]
```